ensime_shared/typecheck.py

```diff
@@ -14,6 +14,13 @@ def buffer_typechecks(self, call_id, payload):
             for note in payload['notes']:
                 self.buffered_notes.append(note)
 
+    def buffer_typechecks_and_display(self, call_id, payload):
+        """Adds typecheck events to the buffer, and displays them right away.
+        This is currently used as a workaround for issue https://github.com/ensime/ensime-server/issues/1616
```

Comment:
Could you please create an issue to track this? if it is a temporary "hack" it should be nice to keep an eye on it and do the right thing once the server issue gets fixed.